modules/serverless-kafka-sasl-public-certs.adoc

```diff
@@ -11,8 +11,9 @@ If you want to use SASL with public CA certificates, you must use the `tls.enabl
 [source,terminal]
 ----
 $ oc create secret --namespace <namespace> generic <kafka_auth_secret> \
+  --from-literal=protocol="SASL_PLAINTEXT" \
   --from-literal=tls.enabled=true \
   --from-literal=password="SecretPassword" \
-  --from-literal=saslType="SCRAM-SHA-512" \
+  --from-literal=sasl.mechanism="SCRAM-SHA-512" \
   --from-literal=user="my-sasl-user"
 ----
```

Comment:
We now ask for a new key called `protocol` in the secret.
Also, key `saslType` is renamed to `sasl.mechanism`.

Reply:
re the comment in the description:

> For the required new keys in the secret (protocol and sasl.mechanism), we are falling back to the old configmap and secret format and infer the values for these keys. However, this fallback code and inferring will be removed in the future. Thus, after the upgrade, our users should update their secrets to have these 2 keys.

Does this mean this needs to change again from the change shown here once a user upgrades to the new implementation?
In this case, I think we should just document this as a step as part of a document / release note about upgrading to the new version and publish that at the same time as any final changes here to the docs.